docker/Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:20.04
+RUN \
+  # configure the "khipster" user
+  groupadd khipster && \
+  useradd jhipster -s /bin/bash -m -g khipster -G sudo && \
```

Comment:
It needs to be changed right to `khipster`?

Reply:
sorry missed that, corrected that